Description: Fix the gemspec
 Don't use git-ls files in Gemspec. There is this warning during the build:
 Invalid gemspec in [puppet-lint.gemspec]: No such file or directory - git
Author: Sophie Brun <sophie@freexian.com>
Last-Update: 2016-09-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/puppet-lint.gemspec
+++ b/puppet-lint.gemspec
@@ -9,9 +9,9 @@ Gem::Specification.new do |s|
   s.description = 'Checks your Puppet manifests against the Puppetlabs
   style guide and alerts you to any discrepancies.'
 
-  s.files = `git ls-files`.split("\n")
-  s.test_files = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  s.files = Dir.glob('**/*')
+  s.test_files = Dir.glob('spec/**/*')
+  s.executables = Dir.glob('bin/*').map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
   s.add_development_dependency 'github_changelog_generator'
